parquet/src/basic.rs

```diff
@@ -215,8 +215,20 @@ pub enum Repetition {
 // Mirrors `parquet::Encoding`
 
 /// Encodings supported by Parquet.
+///
 /// Not all encodings are valid for all types. These enums are also used to specify the
 /// encoding of definition and repetition levels.
+///
+/// Additionally, not all encodings are well supported by the broader ecosystem.
```

Comment:
```suggestion
/// Additionally, not all encodings are well supported by the broader Parquet ecosystem, such as parquet-mr. 
```

Should we call out parquet-mr? In general it would be great to add some specificity if possible (aka what does "broad" mean, can we give examples of widely used systems that only practically support `PLAIN` / `RLE` / `RLE_DICTIONARY`?

Reply:
It is mainly newer systems like DuckDB that have poor support for these encodings, I have opted to reword this slightly to emphasise that the defaults are supported by everything, as opposed to casting aspersions on the other encodings.